src/types.js

```diff
@@ -1,12 +1,72 @@
 export const TYPES = {
   tos: {
-    name: 'Terms of service',
+    name: 'Terms of Service',
     fileName: 'terms_of_service'
   },
+  website_tos: {
+    name: 'Website Terms of Service',
+    fileName: 'website_terms_of_service'
+  },
+  user_agreement: {
+    name: 'User Agreement',
+    fileName: 'user_agreement'
+  },
+  visitor_agreement: {
+    name: 'Visitor Agreement',
+    fileName: 'visitor_agreement'
+  },
+  vendor_tos: {
+    name: 'Vendor Terms of Service',
+    fileName: 'vendor_terms_of_service'
+  },
+  tou: {
+    name: 'Terms of Use',
+    fileName: 'terms_of_use'
+  },
+  acceptableUsePolicy: {
+    name: 'Acceptable Use Policy',
+    fileName: 'acceptable_use_policy'
+  },
+  termsAndConditions: {
+    name: 'Terms and Conditions',
+    fileName: 'terms_and_conditions'
+  },
+  termsAndPrivacyPolicy: {
```

Comment:
We can add this, or we can ask contributors to select the subset of the document which is about ToS and about privacy, especially now that it's been made easier with #70… Could we have some examples of this @michielbdejong? 🙂

Reply:
Good point, and it applies in a few other cases as well. I'll do that.